e2e/teams.spec.ts

```diff
@@ -0,0 +1,211 @@
+import { expect, test } from '@playwright/test'
+import { v4 as uuidv4 } from 'uuid'
+import { testAuth } from './fixtures/authenticatedUserPages'
+import {
+  deleteTeamInvitationMessage,
+  deleteTeamInviteAcceptanceMessage,
+} from './helpers/messages'
+import {
+  getAccessTokenFromCookie,
+  getAdminPAT,
+  getUserIdFromLocalStorage,
+  goToDashboard,
+} from './helpers/testUser'
+import {
+  userConfigs,
+  userPrefix,
+  userValidatedPrefix,
+} from './helpers/userConfig'
+
+const TEAM_NAME = 'swc-e2e-team-' + uuidv4()
+const INVITATION_MESSAGE = 'swc-e2e-invite'
+
+// Run multiple describes in parallel, but run tests inside each describe in order
+// ...tests within describe expect afterAll to be run with the same users, i.e. on the same worker
+// https://playwright.dev/docs/api/class-test#test-describe-configure
+test.describe.configure({ mode: 'serial' })
+
+test.describe('Teams', () => {
+  testAuth(
+    'should exercise team lifecycle',
+    async ({ userPage, validatedUserPage }, testInfo) => {
```

Comment:
Use authenticated user page test fixtures, rather than opening new pages for the test